src/gax/src/error/rpc/mod.rs

```diff
@@ -440,7 +439,7 @@ mod test {
                         ErrorInfo::default()
                             .set_reason("reason")
                             .set_domain("domain")
-                            .set_metadata(HashMap::new()),
+                            .set_metadata([("", "")].into_iter().take(0)),
```

Comment:
Note this change.  `HashMap::new()` becomes ambiguous, there are other ways to fix this (`HashMap::<String, String>::new()` may do), but this is easy enough for a test.